components/playground/main.go

```diff
@@ -82,14 +85,15 @@ if you don't specified a version.
 Examples:
   $ tiup playground nightly                         # Start a TiDB nightly version local cluster
   $ tiup playground v3.0.10 --db 3 --pd 3 --kv 3    # Start a local cluster with 10 nodes
-  $ tiup playground nightly --monitor               # Start a local cluster with monitor system`,
+  $ tiup playground nightly --monitor               # Start a local cluster with monitor system
+  $ tiup playground --pd.config ~/config/pd.toml    # Start a local cluster with specified configuration file`,
 		SilenceUsage: true,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			version := ""
 			if len(args) > 0 {
 				version = args[0]
 			}
-			return bootCluster(version, pdNum, tidbNum, tikvNum, host, monitor)
+			return bootCluster(version, pdConfigPath, dbConfigPath, kvConfigPath, pdNum, tidbNum, tikvNum, host, monitor)
```

Comment:
The signature of function `bootCluster` too complex to hard to maintain, you can define a struct, eg.
```go
type bootOptions {
    version...
}
```

Reply:
OK, bootOptions added.